perun/postprocess/regression_analysis/example/full_computation.perf

```diff
@@ -0,0 +1,322 @@
+      },
+      {
+        "amount": 1,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 9,
+        "uid": "SLList_insert(SLList*, int)"
+      },
+      {
+        "amount": 1,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 10,
+        "uid": "SLList_search(SLList*, int)"
+      },
+      {
+        "amount": 3,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 10,
+        "uid": "SLList_destroy(SLList*)"
+      },
+      {
+        "amount": 16,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 0,
+        "uid": "SLListcls::SLListcls()"
+      },
+      {
+        "amount": 2,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 0,
+        "uid": "SLListcls::Insert(int)"
+      },
+      {
+        "amount": 2,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 1,
+        "uid": "SLListcls::Insert(int)"
+      },
+      {
+        "amount": 2,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 2,
+        "uid": "SLListcls::Insert(int)"
+      },
+      {
+        "amount": 2,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 3,
+        "uid": "SLListcls::Insert(int)"
+      },
+      {
+        "amount": 2,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 4,
+        "uid": "SLListcls::Insert(int)"
+      },
+      {
+        "amount": 2,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 5,
+        "uid": "SLListcls::Insert(int)"
+      },
+      {
+        "amount": 2,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 6,
+        "uid": "SLListcls::Insert(int)"
+      },
+      {
+        "amount": 1,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 7,
+        "uid": "SLListcls::Insert(int)"
+      },
+      {
+        "amount": 1,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 8,
+        "uid": "SLListcls::Insert(int)"
+      },
+      {
+        "amount": 2,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 9,
+        "uid": "SLListcls::Insert(int)"
+      },
+      {
+        "amount": 2,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 10,
+        "uid": "SLListcls::Search(int)"
+      },
+      {
+        "amount": 2,
+        "subtype": "time delta",
+        "type": "mixed",
+        "structure-unit-size": 0,
+        "uid": "SLListcls::~SLListcls()"
+      }
+    ],
+    "time": "0.000165s",
+    "regression_analysis": [
+      {
+        "r_square": 0.08716810077128712,
+        "x_limits": [
```

Comment:
Again, can you shortly elaborate this field? What is included in this list? Are these representation of intervals? If this has known fixed length, I would again consider using dictionary with named fields instead.

Reply:
Changed to x_interval_start and x_interval_end.